JoinColony/colonyCDapp
# fix: update chart UI to match the design

## Description
- [ ] Now that the main content container has increased width, can we add more width to the team labels to reduce the truncation on longer names (I understand there may still be some truncation needed etc).
- [ ] Can we adjust the skeleton state of the team rep widget to show the team labels to be full width? Currently, it's smaller with extra white space on the right side.
- [ ] The team rep component should be fixed height on all team rep versions, no matter how many teams are displayed. Here, a parent team is selected, but the vertical height doesn't match the total in and out component on desktop.

## Testing

1. Check that truncation for long names is reduced: (we shouldn't have a truncation at all because we have limits of up to 20 characters in motion form, but if, in any case, the name is longer, we still cover it) 
<img width="524" alt="image" src="https://github.com/user-attachments/assets/8aa2083e-f76c-4d3b-8290-ff94387cc78a">
2. Check that for reputation with less legends the rep height is still the same as "Total in and out" block:
<img width="774" alt="image" src="https://github.com/user-attachments/assets/9bf0ea25-81e9-4e77-bc8b-ce07559680e0">
3. Go to `src/components/v5/frame/ColonyHome/partials/ReputationChart/ReputationChart.tsx` and update 46 line to true:
`const isDataLoading = true;`
3. Check UI loading state, it should match the design:
<img width="616" alt="image" src="https://github.com/user-attachments/assets/6709d8eb-e410-4707-a6a3-3b2299741149">

Resolves #3209
